deploy/helm/kubecf/assets/operations/instance_groups/credhub.yaml

```diff
@@ -24,16 +25,56 @@
       route_registrar:
         routes:
         - name: credhub
-          tls_port: 8844
+          tls_port: *credhub_port
           server_cert_domain_san: credhub.service.cf.internal
           registration_interval: 10s
           tags:
             component: credhub
           uris:
           - credhub.((system_domain))
-{{- end }}
 
-{{- if not .Values.features.credhub.enabled }}
+- type: replace
+  path: /instance_groups/name=credhub/jobs/-
+  value:
+    name: cf-cli-6-linux
+    release: cf-cli
+    properties:
+      quarks:
+        bpm:
+          processes:
+          - name: credhub-internal-security-group-setup
+            executable: /bin/bash
+            args:
+            - -c
+            - |-
+              {{- .Files.Get "assets/scripts/jobs/cf-cli-6-linux/setup_internal_security_group.sh" | nindent 14 }}
+        envs:
+        - name: CF_CLI_PATH
+          value: /var/vcap/packages/cf-cli-6-linux/bin
+        - name: CF_API
+          value: https://api.((system_domain))
```

Comment:
Should we be using the internal host name for CC for this? (Mostly thinking about things like "what if the external endpoint is behind a load balancer we can't reach from the inside", and "which SSL cert is that going to use")

Reply:
Hmm. I'll try to connect to the internal hostname and load the CA cert to be able to remove `--skip-ssl-validation` from the `cf api` command.

Reply:
Created https://github.com/cloudfoundry-incubator/kubecf/issues/449.